modules/openapi-generator/src/test/java/org/openapitools/codegen/utils/ModelUtilsTest.java

```diff
@@ -26,86 +26,84 @@
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj10B"), "contains 'SomeObj10B'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj11"), "contains 'SomeObj11'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeArrayObj12"), "contains 'SomeArrayObj12'");
-        Assert.assertTrue(allUsedSchemas.contains("ArrayItem12"), "contains 'ArrayItem12'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeArrayObj13"), "contains 'SomeArrayObj13'");
-        Assert.assertTrue(allUsedSchemas.contains("ArrayItem13"), "contains 'ArrayItem13'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj14"), "contains 'SomeObj14'");
-        Assert.assertTrue(allUsedSchemas.contains("PropertyObj14"), "contains 'PropertyObj14'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj15"), "contains 'SomeObj15'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeMapObj16"), "contains 'SomeMapObj16'");
-        Assert.assertTrue(allUsedSchemas.contains("MapItem16"), "contains 'MapItem16'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj17"), "contains 'SomeObj17'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj18"), "contains 'SomeObj18'");
-        Assert.assertTrue(allUsedSchemas.contains("Common18"), "contains 'Common18'");
-        Assert.assertTrue(allUsedSchemas.contains("Obj19ByAge"), "contains 'Obj19ByAge'");
-        Assert.assertTrue(allUsedSchemas.contains("Obj19ByType"), "contains 'Obj19ByType'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj20"), "contains 'SomeObj20'");
-        Assert.assertTrue(allUsedSchemas.contains("OtherObj20"), "contains 'OtherObj20'");
-        Assert.assertTrue(allUsedSchemas.contains("PingDataInput21"), "contains 'PingDataInput21'");
-        Assert.assertTrue(allUsedSchemas.contains("PingDataOutput21"), "contains 'PingDataOutput21'");
-        Assert.assertTrue(allUsedSchemas.contains("SInput22"), "contains 'SInput22'");
-        Assert.assertTrue(allUsedSchemas.contains("SOutput22"), "contains 'SInput22'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeHeader23"), "contains 'SomeHeader23'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeHeader24"), "contains 'SomeHeader24'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj25"), "contains 'SomeObj25'");
-        Assert.assertTrue(allUsedSchemas.contains("SomeObj26"), "contains 'SomeObj26'");
-        Assert.assertTrue(allUsedSchemas.contains("Param27"), "contains 'Param27'");
-        Assert.assertTrue(allUsedSchemas.contains("Param28"), "contains 'Param28'");
-        Assert.assertTrue(allUsedSchemas.contains("Parent30"), "contains 'Parent30'");
-        Assert.assertTrue(allUsedSchemas.contains("AChild30"), "contains 'AChild30'");
-        Assert.assertTrue(allUsedSchemas.contains("BChild30"), "contains 'BChild30'");
+        Assert.assertEquals(allUsedSchemas.size(), 42);
+        List<String> expectedallUsedSchemas = Arrays.asList(
```

Comment:
Be careful when refactoring things like this. While YAML (as used in this method) may process schemas in a consistent order, JSON does not have a determinate order on object properties.

Reply:
Thank you for sharing that json and yaml are processed differently. Do you want me to change this back to our old testing style so it will work if we swap in json in the future?

Reply:
Or optionally I could cast them to sets and check equality of the sets.

Reply:
This check has been changed to use List.containsAll to check presence rather than order.